bundle/regal/lsp/codelens/codelens.rego

```diff
@@ -0,0 +1,82 @@
+# METADATA
+# description: |
+#   the code lens provider decides where code lenses should be placed in the given input file
+# schemas:
+#   - input: schema.regal.ast
+package regal.lsp.codelens
+
+import rego.v1
+
+import data.regal.ast
+import data.regal.result
+import data.regal.util
+
+import data.regal.lsp.util.location
+
+# code lenses are displayed in the order they come back in the returned
+# array, and 'evaluate' somehow feels better to the left of 'debug'
+lenses := array.concat(
+	[l | some l in _eval_lenses],
+	[l | some l in _debug_lenses],
+)
+
+_eval_lenses contains {
+	"range": location.to_range(result.ranged_location_from_text(input["package"]).location),
+	"command": {
+		"title": "Evaluate",
+		"command": "regal.eval",
+		"arguments": [
+			input.regal.file.name,
+			ast.ref_to_string(input["package"].path),
+			util.to_location_object(input["package"].location).row,
+		],
+	},
+}
+
+_eval_lenses contains _rule_lens(rule, "regal.eval", "Evaluate") if {
+	some rule in ast.rules
+}
+
+_debug_lenses contains {
+	"range": location.to_range(result.ranged_location_from_text(input["package"]).location),
+	"command": {
+		"title": "Debug",
```

Comment:
Does vscode support emojis? If it does, we could prefix this with a '🪲' 😄. 
There's probably plenty of setups where that would break, though.

Reply:
It does support that, and since we're only targeting VS Code for now, we could do it. Not sure this is intuitive though. Most people will probably think they got some dirt on their monitor 🤣

<img width="455" alt="bugs" src="https://github.com/user-attachments/assets/3d3445bc-daf2-407a-b628-aecf87aedf24">

Reply:
I was thinking something like `► Evaluate`|`🪲Debug`